PocketExample/Pocket/Models/Transaction.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Transaction.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 4/24/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+
+class Transaction {
+    
+    var nonce: String
+    var recipient: String
+    var value: String
+    var data: NSDictionary
```

Comment:
Use a `Dictionary<>` type or represent it as `[AnyHashable: Any]` or [String: Any] type. More information here: https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/CollectionTypes.html#//apple_ref/doc/uid/TP40014097-CH8-ID113